jupyterhub/services/auth.py

```diff
@@ -731,6 +731,16 @@ def get_current_user(self):
         except Exception:
             self._hub_auth_user_cache = None
             raise
+
+        # store ?token=... tokens passed via url in a cookie for future requests
+        url_token = self.get_argument('token', '')
+        if user_model and url_token \
```

Comment:
Perhaps enclose in parens instead of `\`:
```
if (user_model and url_token
    and getattr(self, '_token_authenticated', False)
    and hasattr(self.hub_auth, 'set_cookie')):
    # comment lines
    self.hub_auth.set_cookie(self, url_token)
```

Reply:
Good call. Done.

Reply:
Thanks. Hard switch back and forth from Dockerfiles and python :wink: